### DISCUSSION TITLE
warp+怎么还是用不了，不是说10月1号就能用吗？

### DISCUSSION THREAD

**Discussion by hdjsjjdgrhj** [share121/inter-knot#5493] [公告]
![Screenshot_2024-10-01-11-02-56-423_com cloudflare onedotonedotonedotone](https://github.com/user-attachments/assets/6a089d50-59eb-48b5-99db-cfc75b33f629)

**Comment by xnpboke** [comment]
多半ip挂了，感觉不如cfwork搭建的节点
